CONTRIBUTING.md

```diff
@@ -0,0 +1,179 @@
+1. Clone the repository locally.
+1. Run `npm install`
+1. Create your `.env` file at the root of the project.
+1. Contact @plocket to get important information for your `.env` file.
+1. Run `npm run setup`
+
+Advanced contributors: If you're going to work on our docassemble internal testing interviews, do so as a contributor on this repository. Avoid forking if you can. The reasons are complex.
+
+## PRs
+
+Before making a pull request
+
+1. Search issues to see if what you are thinking of has been discussed yet.
+1. Make an issue about the change you are thinking of.
+1. Assing yourself to an issue.
+1. Make a new branch.
+1. Make your edits. Feel free to run tests as you go.
+1. When you're ready to push
+   1. Run `npm run test` to make sure your code passes all our tests.
+   1. Make an entry in the `CHANGELOG.md` describing your change.
+   1. AVOID changing the version number. We'll do that after merging your changes.
+   1. If needed, update the `README.md` (e.g. the cheat sheet section) and/or `CONTRIBUTING.md` docs. Changes to our architecture might require editing `CONTRIBUTING.md`.
+1. Push to GitHub.
+1. Wait for the GitHub tests to pass. You may have to re-run failing tests - server restarts or race conditions may interfere with them. We have reduced them greatly, but some are impossible to avoid.
+1. Make your pull request.
+
+## .env
+
+You need to make your own `.env` file. It contains sensitive variables, like passwords. We use them for our internal tests. The file should look something like the below:
+
+```
+# For everyone
+
+DOCASSEMBLE_DEVELOPER_API_KEY=<ask @plocket>
+SERVER_URL=https://apps-dev.suffolklitlab.org
+REPO_URL=https://github.com/SuffolkLITLab/alkiln
+SERVER_RELOAD_TIMEOUT_SECONDS=
+MAX_SECONDS_FOR_SETUP=
+
+# For internal tests
+
+# Set DEBUG to 1 to run in non-headless mode and see the test run live
+DEBUG=
+BRANCH_PATH=v4
+USER1_EMAIL=<ask @plocket>
+USER1_PASSWORD=<ask @plocket>
+SECRET_VAR1=secret-var1-value
+SECRET_VAR2=secret-var2-value
+SECRET_FOR_MISSING_FIELD=secret for missing field
+SECRET_INVALID_THERE_IS_ANOTHER=invalid value for there_is_another
+```
+
+## Running internal tests
+Make the code available to test on the docassemble server:
+```
+npm run setup
+```
+To be able to keep running tests on the same code, avoid takedown.
+
+Use the syntax below to trigger all tests (cucumber and unit tests):
+```
+npm run test
+```
+
+Run only cucumber tests:
+```
+npm run cucumber
+
+# To test Features or Scenarios specific tags:
+npm run cucumber -- "--tags" "@tagname"
+```
+
+To run the unit tests in isolation:
+```
+npm run unit
+```
+
+When done you want to test a new branch or internal test interview files have been updated, clean up before setting up again:
+```
+npm run takedown
+```
+
+# Very general architecture of files and folders
```

Comment:
```suggestion
## Very general architecture of files and folders
```

pages should only have one h1

Reply:
I actually see this as a separate section from the instructions on how to technically interact with our interview. You're right, though. 1 h1. Is it worth shifting everything under this?

Reply:
I agree with the structure over all, just need to bump this and everything under it, yeah.